src/sentry/models/projectteam.py

```diff
@@ -48,23 +47,26 @@ class Meta:
     __repr__ = sane_repr("project_id", "team_id")
 
 
+def process_resource_change(instance, **kwargs):
+    from sentry.tasks.codeowners import update_code_owners_schema
+
+    return (
+        update_code_owners_schema.apply_async(
+            kwargs={
+                "organization": instance.project.organization,
+                "projects": [instance.project],
+            }
+        ),
```

Comment:
Is this intentionally returning a 1-tuple?

Reply:
yes